### DISCUSSION TITLE
- Final Project (Work-in-Progress II)

### DISCUSSION THREAD

**Discussion by giscus** [DACSS/DACSS_601_August2022_v2#257] [Announcements]
# - Final Project (Work-in-Progress II)

Analyzing Snapchat Political Ad Targeting and Expenditure in the US in 2020

https://dacss.github.io/DACSS_601_August2022_v2/posts/FinalProject_AnanyaPujary.html

**Comment by LarriMill** [comment]
Hi Ananya, this is nicely done overall. I really loved that you gave your readers outside information, footnotes, and clear vocab definitions. Your map graphs were beautiful, but make sure that all of your graphs are clearly described with titles and descriptive labels. Also, it's better not have your reader scroll through thousands of rows of data-- if I weren't grading, I likely would have given up, without seeing all of your hard work at the end.
